Cargo.toml

```diff
@@ -21,35 +20,294 @@ homepage = "https://www.rusoto.org/"
 travis-ci = { repository = "rusoto/rusoto", branch = "master" }
 appveyor = { repository = "rusoto/rusoto", branch = "master" }
 
-[build-dependencies]
-rayon = "0.6.0"
+[dependencies.rusoto_core]
+path = "rusoto/core"
+version = "0.24.0"
 
-[build-dependencies.rusoto_codegen]
-default-features = false
-path = "codegen"
-version = ">= 0.8.0"
+[dependencies.rusoto_acm]
+optional = true
+path = "rusoto/services/acm"
+version = "0.24.0"
 
-[dependencies]
-chrono = "0.2.21"
-hyper = "0.10.0"
-log = "0.3.6"
-md5 = "0.3.2"
-rustc-serialize = "0.3.19"
-serde = "0.9.5"
-serde_derive = "0.9.5"
-serde_json = "0.9.4"
-xml-rs = "0.3"
+[dependencies.rusoto_autoscaling]
+optional = true
+path = "rusoto/services/autoscaling"
+version = "0.24.0"
 
-[dependencies.rusoto_core]
-path = "rusoto/core"
+[dependencies.rusoto_cloudformation]
+optional = true
```

Comment:
D'oh.  This little detail was a big piece of what I was missing conceptually when I tried to get this split out but still using the same workspace.  It never even occurred to me to declare the sub-crate dependencies optional.  Turns out that NOT doing that makes this mostly unworkable.